### PR TITLE
Added the new telemetry syntax to the following packages, which are all that had previously telemetry implemented: 

### PR DESCRIPTION
**Pull request by igor-ribeiiro**
Added the new telemetry syntax to the following packages, which are all that had previously telemetry implemented: 
- office-addin-debugging 
- office-addin-dev-certs 
- office-addin-lint 
- office-addin-manifest 
- office-addin-sso

### CODE REVIEW

File: packages/office-addin-lint/src/lint.ts
Comment (akrantz) [inline packages/office-addin-lint/src/lint.ts:40]: Why is this string "Has lint error"? Sorry I didn't look at the code and cannot tell here when looking at the diff.

Comment (igor-ribeiiro) [reply packages/office-addin-lint/src/lint.ts:40]: To send an "expectedException", it is required to pass the ```err``` parameter to the functions.
So I passed this string to represent the expected error.

**Review by akrantz — COMMENTED** [review]
Looks good overall. It is interesting that there was a use case to call reportExpectedException(). 

I'm curious about how it is used with regard to the second string "Has lint problem" or "Has formatting issue". Just wondering if that string should be different or not...

**Comment by igor-ribeiiro** [comment]
> Looks good overall. It is interesting that there was a use case to call reportExpectedException().
> 
> I'm curious about how it is used with regard to the second string "Has lint problem" or "Has formatting issue". Just wondering if that string should be different or not...

I also liked to see that the code already had a use case for the  ```reportExpectedException``` function.
This string parameter will be the error message that will show on the logs.

So, when looking through the logs, you could see that is had ```Succeed = true```, while being an error. Then you could look into this message and see why this expected error happened.

File: packages/office-addin-lint/src/lint.ts
Comment (akrantz) [reply packages/office-addin-lint/src/lint.ts:40]: Based on the function definitiion:
`public reportExpectedException(method: string, err: Error | string, data: object = {}): void`

I would expect the second parameter to be `err` and not `"Has lint error"`. The exitCode actually distinguishes the cases because exitCode 1 means there is a lint error and 2 for unexpected error.

Let me know if I'm missing something.

Comment (igor-ribeiiro) [reply packages/office-addin-lint/src/lint.ts]: Sure, a mistake by me. Err should be passed instead of the string

**Comment by akrantz** [comment]
@igor-ribeiiro Does this remove all existing usage of the deprecated usage data methods now?

**Review by akrantz — APPROVED** [review]
Looks good to me. Should probably wait for approval from @millerds too before merging.

**Comment by igor-ribeiiro** [comment]
> @igor-ribeiiro Does this remove all existing usage of the deprecated usage data methods now?

Was missing some tests in the usage-data package. Now, there are 0 uses of the deprecated functions within this repository.

**Review by akrantz — APPROVED** [review]
I'd like to have @millerds also approve. 

A tip when you squash: you can edit the commit message and copy the PR description rather than using the default text where it puts the individual commit text (which isn't useful here).

**Review by millerds — APPROVED** [review]

**Pull request merged**
